spec/support/helpers/authentication_helper.rb

```diff
@@ -1,5 +1,6 @@
 module AuthenticationHelper
   def sign_in_as(user)
+    visit root_path
```

Comment:
Instead of visiting root path and then clicking on a sign in link, why not just visit the sign in path directly? Making as few movements across the site as possible will help keep your tests speedier